workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -49,35 +48,6 @@
 gravity = 9.81
 
 
-def setup_metrics_logger():
```

Comment:
logging functions moved to `logs.py` (which also includes stuff that was previously in `capture.py`)

Reply:
Thank you, this is a great cleanup!